test/content-legacy/no-content.html

```diff
@@ -53,17 +53,18 @@
                 errorSpy.restore();
             });
             before(function () {
-                var server = sinon.fakeServer.create();
+                server = sinon.fakeServer.create();
                 server.xhr.useFilters = true;
                 server.xhr.addFilter(function (method, url) {
-                    //whenever the this returns true the request will not faked
-                    return !url.match(/mock\//);
+                    //whenever this returns true the request will not faked
+                    return !url.match(/mock\/204/);
                 });
                 server.respondWith('GET', '/mock/204', [204, {"Content-Type": "text/html"}, '']);
```

Comment:
Maybe change the responses generated by sinon to be `/generate/204` instead of `/mock/204`? This will make it easier to distinguish.

Or rename the `mock` directory to `static`.